app/black_spots/tasks/load_road_network.py

```diff
@@ -0,0 +1,53 @@
+import glob
+import os
+import shutil
+import subprocess
+
+from django.conf import settings
+
+from celery import shared_task
+from celery.utils.log import get_task_logger
+
+import requests
+
+
+logger = get_task_logger(__name__)
+
+
+@shared_task
+def load_road_network(output_root=None, output_srid='EPSG:3395'):
+    """Downloads OpenStreetMap data to a Shapefile by inspecting uploaded boundaries
```

Comment:
This should be updated now that grabbing the url via boundary inspection isn't possible.